components/properties/src/maps.rs

```diff
@@ -62,6 +62,26 @@ where
     get_cp_map(provider, key::GENERAL_CATEGORY_V1)
 }
 
+/// Return a [`CodePointTrie`] for the BidiClass Unicode enumerated property. See [`BidiClass`].
```

Comment:
The Unicode property official aliases are `bc` and `Bidi_Class`. `BidiClass` is the Rust enum.
```suggestion
/// Return a [`CodePointTrie`] for the Bidi_Class Unicode enumerated property. See [`BidiClass`].
```